### PR TITLE
Fix more warnings

### PR DESCRIPTION
**Pull request by Morriar**
Everything is in the title, for the algorithm see #2027.

Spam box: @R4PaSs @xymus @privat and even @BlackMinou for ballz

### CODE REVIEW

**Comment by privat** [comment]
+1
great troll

File: contrib/inkscape_tools/tests/app/src/s2pn.nit [hunk 1]
```diff
@@ -22,27 +22,24 @@ redef class App
 
 	init do super
 
-	redef fun on_create
-	do
+	redef fun on_create do
```
Comment (xymus) [inline contrib/inkscape_tools/tests/app/src/s2pn.nit]: Fixing whitespaces is fine with me, but please preserve the coding style of the inkscape_tools project. The `do`s should stay on the next line.

**Comment by xymus** [comment]
+1 when the style is fixed in inkscape_tools!

**Comment by Morriar** [comment]
Reroll without the troll, I guess I have the +1 from @xymus now!

**Pull request merged**